server/api-service/lowcoder-server/src/test/resources/application.yml

```diff
@@ -17,4 +21,13 @@ demo:
       port: 3306
       timeout: 100
       username: 123
-      host: localhost
\ No newline at end of file
+      host: localhost
+
+common:
+  cookie-name: UT-TACO-TOKEN
+
+auth:
+  email:
+    enable: true
+  google:
```

Comment:
We do not have a google Auth yet. Does this setting makes sense?

Reply:
You are right. My bad.
I added it initially when working on testGoogleRegisterSuccess(), forgot to remove. 
Will remove the unneeded config